pkgs/tools/misc/cicero-tui/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, cmake
+, pkg-config
+, expat
+, freetype
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cicero-tui";
+  version = "0.1.4";
+
+  src = fetchFromGitHub {
+    owner = "eyeplum";
+    repo = "cicero-tui";
+    rev = "v${version}";
+    sha256 = "1bz2y37qf9c3fxc73chb42rffdivp5krczhgd9rnwq5r6n6bdgq7";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ];
+  buildInputs = [
```

Comment:
According to their readme build inputs are wrong https://github.com/eyeplum/cicero-tui#fontconfig-and-freetype

Reply:
I didn't see any change in the functionnalities but now cicero-tui is linking to libfontconfig.